src/main/resources/application.yml

```diff
@@ -33,6 +33,7 @@ jobs:
     name: "BatchPrintRequests"
     cron: "0 0/15 * * * *" # every 15 minutes, starting on the hour - see analysis and recommendations in EIP1-2515
     batch-size: 50
+    daily-limit: 150000
```

Comment:
Minor (and I can live without this if its not possible ... I'm not sure if it is even possible ....)  ... do Spring properties allow to you express numbers with comma place delimiters? IE. can you do this?
```
daily-limit: 150,000
```
The only reason I mention it is that I'm currently viewing this on a smaller screen. Where I'm in my more "senior years" 😁 my eyesight is not what it used to be (even with my glasses on!), and I had to zoom the screen to determine that it was 150K rather than 15K !! 🤣 

(God, I'm so blind! 🤣 )

Reply:
Thanks, for the suggestion @NathanRussellValtech. 
I will definitely look into it to make it more readable if possible.

Reply:
@NathanRussellValtech , seems like we can do this:

<pre>
daily-limit: 150_000
</pre>

Reply:
Awesome! Hurrah for Spring being able to cater for my old man eyes! 🤣 🎉